src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1498,7 +1500,10 @@ void S6Exporter::ExportTileElement(RCT12TileElement* dst, TileElement* src)
             auto dst2 = dst->AsTrack();
             auto src2 = src->AsTrack();
 
-            dst2->SetTrackType(src2->GetTrackType());
+            uint16_t trackType = src2->GetTrackType();
+            if (trackType == TrackElemType::RotationControlToggle)
+                trackType = TrackElemType::RotationControlToggleAlias;
+            dst2->SetTrackType(trackType);
```

Comment:
how is this not causing a warning. You need to convert to a uint8_t here. I would make it explicit add:
```
// SV6 track type is uint8_t we have aliased rotation control to booster to allow this
dst2->SetTrackType(static_cast<uint8_t>(trackType));
```
Also can you make it:
```
auto trackType = src2->GetTrackType();
```
Then when we make that function return a `track_type_t` no changes will be required.